sql/colflow: TestVectorizedFlowDeadlocksWhenSpilling failed

sql/colflow.TestVectorizedFlowDeadlocksWhenSpilling [failed](https://tanzanite.cluster.engflow.com/invocations/default/11cd5bc0-d521-4a19-a0a6-183bc6238f94?testReportRun=28&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc3FsL2NvbGZsb3c6Y29sZmxvd190ZXN0) on master @ [4ee917141b193bd2db451258cd040dc3b423951e](https://github.com/cockroachdb/cockroach/commits/4ee917141b193bd2db451258cd040dc3b423951e):


```
=== RUN   TestVectorizedFlowDeadlocksWhenSpilling
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestVectorizedFlowDeadlocksWhenSpilling917547535
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vectorized_flow_deadlock_test.go:77: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/sql/colflow_test/pkg/sql/colflow/vectorized_flow_deadlock_test.go:77
        	Error:      	Should be false
        	Test:       	TestVectorizedFlowDeadlocksWhenSpilling
    panic.go:523: -- test log scope end --
test logs left over in: outputs.zip/logTestVectorizedFlowDeadlocksWhenSpilling917547535
--- FAIL: TestVectorizedFlowDeadlocksWhenSpilling (12.92s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=28</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestVectorizedFlowDeadlocksWhenSpilling.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-34570

--- a/pkg/sql/colflow/vectorized_flow_deadlock_test.go
+++ b/pkg/sql/colflow/vectorized_flow_deadlock_test.go
@@ -62,7 +62,7 @@ func TestVectorizedFlowDeadlocksWhenSpilling(t *testing.T) {
 	_, err = conn.ExecContext(ctx, "SET CLUSTER SETTING sql.distsql.acquire_vec_fds.max_retries = 1")
 	require.NoError(t, err)
 
-	queryCtx, queryCtxCancel := context.WithDeadline(ctx, timeutil.Now().Add(10*time.Second))
+	queryCtx, queryCtxCancel := context.WithDeadline(ctx, timeutil.Now().Add(time.Minute))
 	defer queryCtxCancel()
 	// Run a query with a hash joiner feeding into a hash aggregator, with both
 	// operators spilling to disk. We expect that the hash aggregator won't be
@@ -74,5 +74,5 @@ func TestVectorizedFlowDeadlocksWhenSpilling(t *testing.T) {
 	// We expect an error that is different from the query cancellation (which
 	// is what SQL layer returns on a context cancellation).
 	require.NotNil(t, err)
-	require.False(t, strings.Contains(err.Error(), cancelchecker.QueryCanceledError.Error()))
+	require.False(t, strings.Contains(err.Error(), cancelchecker.QueryCanceledError.Error()), err)
 }

